docs/ai/quickstarts/chat-local-model.md

```diff
@@ -14,9 +14,9 @@ In this quickstart, you learn how to create a conversational .NET console chat a
 
 ## Prerequisites
 
-* [Install .NET 8.0](https://dotnet.microsoft.com/download) or higher
+[!INCLUDE [Prerequisites](../../../includes/dotnet-prerequisites.md)]
```

Comment:
This should probably be inserted after any existing bullets - otherwise the last bullet is [lost at the end](https://review.learn.microsoft.com/en-us/dotnet/ai/quickstarts/chat-local-model?branch=pr-en-us-45351#prerequisites):

<img width="455" alt="image" src="https://github.com/user-attachments/assets/054e3a63-e985-494b-b35d-4f77675c0c1b" />